conf/machine/include/imx-base.inc

```diff
@@ -400,6 +406,8 @@ SOC_DEFAULT_WKS_FILE_mxs ?= "imx-uboot-mxs-bootpart.wks.in"
 # WKS file, as it packs the flash.bin binary into the boot partition.
 # This binary is produced by U-Boot build itself, and is serves as a
 # direct replacement of imx-boot from NXP.
+# Creation of the flash.bit is controlled by UBOOT_PROVIDES_BOOT_CONTAINER
```

Comment:
I guess that would be s/flash.bit/flash.bin/

Reply:
Thanks, typos are evil! 😉